source/d2sqlite3/database.d

```diff
@@ -56,6 +56,27 @@ enum Deterministic
     no = 0
 }
 
+struct PrepareMany {
+	Database db;
+	Statement current;
+	string sql_left;
+	bool empty;
+	this(Database db, const string sql) {
+		this.db = db;
+		this.sql_left = sql;
+		popFront();
+	}
+	void popFront() {
+		const size_t old = sql_left.length;
+		current = Statement(this.db, sql_left);
+		empty = old == sql_left.length;
+		sql_left = sql_left.chomp();
+	}
+	Statement front() {
+		return current;
+	}
+}
+
```

Comment:
This is lacking visibility attributes (`private` / `package`) and attributes (`@safe` / `nothrow` / ...)